dd-java-agent/instrumentation/opentracing/api-0.31/src/main/java/datadog/trace/instrumentation/opentracing31/OTScopeManager.java

```diff
@@ -67,4 +76,33 @@ public boolean isFinishSpanOnClose() {
       return finishSpanOnClose;
     }
   }
+
+  private final class FakeScope implements AgentScope {
```

Comment:
nitpick: It is really a fake? It will still operate most of the time isn’t it?
```suggestion
  private final class CapturedActivedScope implements AgentScope {
```

Reply:
This is a scope that didn't come from the scope manager - ideally never used, only there to satisfy a long deprecated API. So I wanted a name that stood out and couldn't be confused with other names in the codebase.